matplotlibrc.template

```diff
@@ -55,61 +92,63 @@
 ## When True, open the webbrowser to the plot that is shown
 #webagg.open_in_browser : True
 
-## if you are running pyplot inside a GUI and your backend choice
+## If you are running pyplot inside a GUI and your backend choice
 ## conflicts, we will automatically try to find a compatible one for
 ## you if backend_fallback is True
-#backend_fallback: True
+#backend_fallback : True
 
-#interactive  : False
-#toolbar      : toolbar2   ## None | toolbar2  ("classic" is deprecated)
-#timezone     : UTC        ## a pytz timezone string, e.g., US/Central or Europe/Paris
+#interactive : False
+#toolbar     : toolbar2  ## None | toolbar2  ("classic" is deprecated)
+#timezone    : UTC       ## a pytz timezone string, e.g., US/Central or Europe/Paris
 
 ## Where your matplotlib data lives if you installed to a non-default
 ## location.  This is where the matplotlib fonts, bitmaps, etc reside
 #datapath : /home/jdhunter/mpldata
 
 
 #### LINES
```

Comment:
The file is only plain text and has no formal format. Using sections in markdown-like style does not stand out enough. I suggest using something like
~~~
## *** LINES *****************************************************************
~~~
or even
~~~
## ***************************************************************************
## * LINES                                                                   *
## ***************************************************************************
~~~
for sectioning.

Reply:
Great idea! And thanks for the reviewing!